### PR TITLE
create collection

### PR DESCRIPTION
**Pull request by asiniscalchi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Review the [FRAME runtime implementation](./runtime/src/lib.rs) included in this
 
 ### Pallets
 
-The runtime in this project is constructed using many FRAME pallets that ship with the [core Substrate repository](https://github.com/paritytech/substrate/tree/master/frame) and a template pallet that is [defined in the `pallets`](./pallets/template/src/lib.rs) directory.
+The runtime in this project is constructed using many FRAME pallets that ship with the [core Substrate repository](https://github.com/paritytech/substrate/tree/master/frame) and a template pallet that is [defined in the `pallets`](./pallets/living-assets-evolution/src/lib.rs) directory.
```
Comment (dastanfv) [inline README.md:134]:
```suggestion
The runtime in this project is constructed using many FRAME pallets that ship with the [core Substrate repository](https://github.com/paritytech/substrate/tree/master/frame) and a pallet that is [defined in the `pallets`](./pallets/living-assets-evolution/src/lib.rs) directory.
```

Comment (dastanfv) [reply README.md:134]: I think generally this part of the doc should be removed, it doesn't add anything meaningful and doesn't belong to our project

File: pallets/living-assets-evolution/src/lib.rs [hunk 2]
```diff
@@ -34,75 +38,80 @@ pub mod pallet {
...
 	// https://docs.substrate.io/main-docs/build/events-errors/
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		/// Event documentation should end with an array that provides descriptive names for event
-		/// parameters. [something, who]
-		SomethingStored { something: u32, who: T::AccountId },
+		/// Collection created
+		/// parameters. [collection_id, who]
+		CollectionCreated { collection_id: CollectionId, who: T::AccountId },
```
Comment (dastanfv) [inline pallets/living-assets-evolution/src/lib.rs:59]:
```suggestion
		CollectionCreated { collection_id: CollectionId, owner: T::AccountId },
```

File: pallets/living-assets-evolution/src/tests.rs [hunk 3]
```diff
@@ -0,0 +1,45 @@
+use crate::{mock::*, CollectionId, Event};
+use frame_support::assert_ok;
+
+#[test]
+fn owner_of_unexistent_collection() {
```
Comment (dastanfv) [inline pallets/living-assets-evolution/src/tests.rs:5]:
```suggestion
fn owner_of_inexistent_collection() {
```

File: pallets/living-assets-evolution/src/tests.rs [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+fn counter_of_collection_increases() {
+	new_test_ext().execute_with(|| {
+		assert_eq!(TemplateModule::collection_counter(), 0);
+		assert_ok!(TemplateModule::create_collection(RuntimeOrigin::signed(1)));
+		assert_eq!(TemplateModule::collection_counter(), 1);
+	})
+}
+
+#[test]
+fn crete_collection_emits_event() {
```
Comment (dastanfv) [inline pallets/living-assets-evolution/src/tests.rs:36]:
```suggestion
fn create_collection_emits_event() {
```

File: README.md (hunk 1 above)
Comment (asiniscalchi) [reply README.md]: I agree ... I will create a PR to rewrite the README: https://github.com/freeverseio/laos-evolution-node/pull/75

**Review by dastanfv — APPROVED** [review]

**Pull request merged**